addons/sale_product_configurator/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -285,7 +285,10 @@ export class ProductConfiguratorDialog extends Component {
         if (parentCombination) {
             for(const ptavId of parentCombination) {
                 for(const excludedPtavId of (parentExclusions[ptavId]||[])) {
-                    ptavList.find(ptav => ptav.id === excludedPtavId).excluded = true;
```

Comment:
Can't we avoid sending disabled attributes from the server instead ?

(please also add the ticket number to your commit)

Reply:
Actually, we can from [here](https://github.com/odoo/odoo/blob/17.0/addons/product/models/product_template.py#L975C21-L975C83). 
But it may affect the creation/selection of product variants. 

The fix was done to imitate, how it worked in previous version. 
previously, the unwanted combinations would be disabled using jquery. 
something like [this](https://github.com/odoo/odoo/blob/16.0/addons/sale/static/src/js/variant_mixin.js#L405-L411)
the [_disableInput](https://github.com/odoo/odoo/blob/16.0/addons/sale/static/src/js/variant_mixin.js#L505)  function silently fail if corresponding elements were not found/unavailable.

Reply:
Indeed, it can be safer to handle it here for now :ok_hand:.